system/handlers/formcontrols/DerivativePicker.cfc

```diff
@@ -0,0 +1,19 @@
+component output=false {
+
+	property name="assetManagerService"  inject="AssetManagerService";
+
+	public string function index( event, rc, prc, args={} ) output=false {
+
+		var derivatives = assetManagerService.listEditorDerivates();
+		args.labels = [];
```

Comment:
We need to return an empty string here in the case where there are no derivatives, so that the form field doesn't show. So:

    if ( !derivatives.len() ) {
        return "";
    }